changelog.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [v3.4.0]
+### Added
+* Be able to specific a desired video resolution with `createMedia`
+* Default video resolution to highest possible (up to 4k)
```

Comment:
Are there any doc changes that need to be made?

Reply:
good call.